debug.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]; then
+	cat <<USAGE
+Usage: $0 BROWSER JENKINS [ARGS]
+
+The script runs dryrun tests first to discover trivial problems immediately.
+It can use jenkins.war from local maven repository or download it when missing.
+
+BROWSER: Value for BROWSER variable
+JENKINS: Path to the jenkins.war, Jenkins version of one of "latest", "latest-rc", "lts" and "lts-rc"
+
+Examples:
+
+# Run full suite in FF against ./jenkins.war.
+$ ./run firefox ./jenkins.war
+
+# Run Ant plugin test in chrome against Jenkins 1.512.
+$ ./run chrome 1.512 -Dtest=AntPluginTest
+
+# Run full suite in FF against LTS release candidate
+$ ./run firefox lts-rc
+USAGE
+  exit -2
+fi
+
+browser=$1
+war=$2
+if [ ! -f $war ]; then
+    mirrors=http://mirrors.jenkins-ci.org
```

Comment:
It would be better to download from the main repo using HTTPs. Mirrors are flaky sometimes